packages/blockchain-fabric-admin/src/LifecyclePeer.ts

```diff
@@ -174,6 +207,52 @@ export class LifecyclePeer {
         }
     }
 
+    public async getAllInstalledSmartContractsV1(requestTimeout?: number): Promise<InstalledSmartContract[]> {
```

Comment:
As `blockchain-fabric-admin` is for the v2 lifecycle, I think we should probably move this function out into a new package.

Reply:
hmm.. We could argue fabric admin is an admin package for the v2 sdk, which handles all allowed admin functionality, which includes v1 and v2. I can create a new package, and we might need to do it in the future, but for now there is perhaps one or two functions to go in it, and it would probably require re-defining structures or depend on fabric-admin. Is this worth doing now or should we make this decision when more functionality is merged?